README.md

```diff
@@ -290,12 +290,12 @@ function* handleFetchLandAmountRequest(action: FetchLandAmountRequestAction) {
 
 ## Storage
 
-The storage module allows you to save parts of the redux store in localStorage to make them persistent.
-This module is required to use other modules like `Transaction` and `Translation`.
+The storage module allows you to save parts of the redux store in localStorage to make them persistent and migrate it from different versions without loosing it.
+This module is required to use other modules like `Transaction`, `Translation`, `Wallet` and `Storage`.
```

Comment:
The `Storage` module is irrelevant in this enumeration (this is already the `Storage` module)